functions/auth.js

```diff
@@ -6,9 +6,14 @@ const { HttpsError } = require("firebase-functions/lib/providers/https");
 const { fetchAuthConfig } = require("./src/api/auth");
 const { addAdmin } = require("./src/api/roles");
 
-const { assertValidUrl, assertValidVenueId } = require("./src/utils/assert");
+const {
+  assertValidUrl,
+  assertValidVenueId,
+  checkAuth,
+} = require("./src/utils/assert");
 const { createOAuth2Client } = require("./src/utils/auth");
 const { getJson, postJson } = require("./src/utils/fetch");
+const { chunk } = require("lodash");
```

Comment:
@ermiaSparkle this import should be at the top, for my opinion

Reply:
fixed in 5de97cde144760c02d63d94cad7ef3f213a5d232